### DISCUSSION TITLE
[question]请问有高可用集群部署建议吗？

### DISCUSSION THREAD

**Discussion by brzyangg** [bfenetworks/bfe#824] [Ideas（新功能的建议和讨论）]
请问有高可用集群部署建议吗？

想部署多个bfe实例，搭建bfe集群

主要问题：

*  配置应该如何管理，目前我看都是使用配置文件，如果部署多台机器能否集中管理

**Comment by iyangsj** [comment]
感谢关注！有两种方式：
1. 使用方基于自身已有的基础设施，实现配置的管理及下发
2. BFE开源项目计划在H2发布控制平台（后续也可基于该控制平台集中管理配置）

**Comment by brzyangg** [comment]
感谢回复！

**Comment by skycyan** [comment]
> 感谢关注！有两种方式：
> 
> 1. 使用方基于自身已有的基础设施，实现配置的管理及下发
> 2. BFE开源项目计划在H2发布控制平台（也可基于控制平台集中管理配置）

===========
点赞，增加控制平台，有配置管理功能后，可用性更高。
